### PR TITLE
Rental

### PR DESCRIPTION
**Pull request by GuruKiranDN**
issues 
Logo image
content image
media query(responsivity)
animation

### CODE REVIEW

File: pages/rental.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
+import React,{Component} from 'react';
```
Comment (ankitc1010) [inline pages/rental.js]: This should be there in ```components/page/Rental/index.js````

File: pages/rental.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+      </Body>
+     
+      );
+    }
+  }
+ export default rental;
+
+
+
+ const Body = sc.body`
```
Comment (ankitc1010) [inline pages/rental.js]: Create `one` styled component, rather than creating so many styled components.

File: pages/rental.js (hunk 1 above)
Comment (GuruKiranDN) [reply pages/rental.js]: have moved as above in the latest commit but not able to run the code

File: components/pages/Rental/index.js [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+      </Body>
+     
+      );
+    }
+  }
+ export default rental;
+
+
+
+ const Body = sc.body`
```
Comment (ankitc1010) [inline components/pages/Rental/index.js]: Delete Them.

Comment (GuruKiranDN) [reply components/pages/Rental/index.js]: what exactly ?

**Pull request merged**